app/views/conferences/_conference_details.html.haml

```diff
@@ -38,3 +38,5 @@
               = link_to 'Subscribe', conference_subscriptions_path(conference.short_title), method: :post, class: 'btn btn-default'
             - else
               = link_to 'Unsubscribe', conference_subscriptions_path(conference.short_title), method: :delete, class: 'btn btn-default'
+            - if !current_user.nil? && current_user.tickets(conference).any?
```

Comment:
Would be better to use **present?** here? Also if you use the scoped relationship for **physical_tickets** it will look like this:

```ruby
if current_user.present? && current_user.physical_tickets(conference).any?
```
Or just using **try**:

```ruby
if current_user.try(:physical_tickets, conference).try(:any?)
```